src/SpectatorView.Unity/Assets/SpectatorView/Scenes/SpectatorView.TestScene1.unity

```diff
@@ -0,0 +1,361 @@
+%YAML 1.1
```

Comment:
Are these test scenes necessary for the primary SpectatorView project, or could they be included under samples/SpectatorView.Example.Unity?

Reply:
They can be moved to SpectatorView.Example.Unity